bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -264,6 +265,15 @@
    */
   public static final String BIGTABLE_RPC_TIMEOUT_MS_KEY = "google.bigtable.rpc.timeout.ms";
 
+  /**
+   * If timeouts are set, how many milliseconds should pass before a DEADLINE_EXCEEDED for an
+   * individual RPC attempt? Note that multiple attempts may happen within an overall operation,
+   * whose timeout is governed by {@link #BIGTABLE_RPC_TIMEOUT_MS_KEY}. Currently, this feature is
+   * experimental.
+   */
+  public static final String BIGTABLE_RPC_ATTEMPT_TIMEOUT_MS_KEY =
+      "google.bigtable.rpc.attempt_timeout.ms";
```

Comment:
To follow the naming convention, should it be attempt.timeout?

Reply:
+1

Reply:
Done